manifests/kustomize/README.md

```diff
@@ -81,7 +75,25 @@ Or you can add it in one line with:
 
 ```bash
 kubectl get configmap centraldashboard-config -n kubeflow -o json | jq '.data.links |= (fromjson | .menuLinks += [{"icon": "assignment", "link": "/model-registry/", "text": "Model Registry", "type": "item"}] | tojson)' | kubectl apply -f - -n kubeflow
-````
+```
+
+#### Installing Model Registry on a Kubeflow Profile
```

Comment:
can I also say "Installing Model Registry on a Kubeflow Profile (separate namespace)"?

Reply:
Yes, that might be helpful